src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -16,6 +16,8 @@
 
 namespace System.Runtime.Loader
 {
+    public delegate Assembly ResolveEventHandler(object sender, AssemblyName assembly);
```

Comment:
I am not sure whether it is a good thing that it is the same name, but different signature and namespace as what's in full .NET Framework. The fx design folks should chime in about the name.